### ISSUE TITLE
Client API missing DeleteKey

### ISSUE THREAD

**Issue by LeonOnRoad** [RedisTimeSeries/redistimeseries-go#85]
Hi,
I observed that the API of redis-timeseries client doesn't have the functionality to delete a key.
I want to avoid using 2 redis clients in my code (this client and "github.com/go-redis/redis")
Can you please add it?

My business case: I create a timeseries for a key, store samples, do aggregations and then delete key.

Thanks,
Cristian

**Comment by LeonOnRoad** [comment]
For the moment I implemented the following workaround in my code:
```go
func DeleteKey(client *redistimeseries.Client, key string) error {
	conn := client.Pool.Get()
	defer conn.Close()

	_, err := conn.Do("DEL", key)

	return err
}
```

**Comment by filipecosta90** [comment]
hi there @LeonOnRoad , #86 address it ( PS: feel free to open PRs when you have code portions like the above... I basically copy pasted yours, just changed the name of the function (given that DeleteKey IMHO is to much generic ) and added the example ). 


Even though `DeleteSerie` basically being sugar coating on the go client that calls DEL <ts> command I tend to agree that having a documented and exemplified DeleteSerie will make the usage more clearer. 

Wanted to confirm with @danni-m what he thinks about it.

**Comment by LeonOnRoad** [comment]
Hi @filipecosta90 ,
Thanks for accepting my change request. That was super fast! 👍 
I agree on the naming.

LGTM

This issue can be closed. Thanks!

**Comment by filipecosta90** [comment]
@LeonOnRoad we've just released it on [1.4.3](https://github.com/RedisTimeSeries/redistimeseries-go/releases/tag/v1.4.3)
Thank you for bringing this subject up.
